crates/sui-graphql-rpc/src/server/version.rs

```diff
@@ -41,7 +41,7 @@ impl headers::Header for SuiRpcVersion {
     }
 
     fn encode<E: Extend<HeaderValue>>(&self, _values: &mut E) {
-        unimplemented!()
+        // unimplemented!()
```

Comment:
Ditto -- we want this one, because we shouldn't be encoding this header.

Reply:
revived